mcstatus/__main__.py

```diff
@@ -42,12 +41,18 @@ def json(server: JavaServer) -> None:
         if status_res.players.sample is not None:
             data["players"] = [{"name": player.name, "id": player.id} for player in status_res.players.sample]
 
+        #ping_res = server.ping() # status calls test_ping(), a 2nd ping is not needed.
+        #data["ping"] = ping_res
```

Comment:
No need for explanations here, accessing `status_res.latency` is descriptive enough.
```suggestion
```